front/app/containers/Admin/dashboard/overview/index.tsx

```diff
@@ -144,16 +136,11 @@ const OverviewDashboard = ({ projects }: DataProps) => {
           />
         </Column>
         <Column>
-          <BarChartActiveUsersByTime
-            graphUnit="users"
-            graphUnitMessageKey="activeUsers"
-            graphTitle={formatMessage(messages.participants)}
-            xlsxEndpoint={activeUsersByTimeXlsxEndpoint}
-            infoMessage={formatMessage(
-              messages.numberOfActiveParticipantsDescription
-            )}
-            className="e2e-users-by-time-cumulative-chart fullWidth"
-            {...legacyProps}
+          <ActiveUsersCard
+            projectId={currentProjectFilter}
+            startAtMoment={startAtMoment}
+            endAtMoment={endAtMoment}
+            resolution={resolution}
```

Comment:
Replaced the old 'Active users' graph by the 'Participants' graph from the overview dashboard (still confusingly named here, will be fixed later)

<img width="581" alt="Screenshot 2024-04-16 at 20 36 08" src="https://github.com/CitizenLabDotCo/citizenlab/assets/13822005/9f5a031b-561b-4396-9286-196e3a0a06fd">

Reply:
Also will still tweak this a bit in a following PR (probably remove the statistics)